src/main/resources/db/migration/V1_69__alter_content_store.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE juror_mod.content_store
+	ADD PRIMARY KEY (request_id),
+	ADD column failed_file_transfer boolean null;
```

Comment:
As this allows nulls - will there be any migration impact? Or are we happy to migrate all heritage values without populating this column?

Reply:
By default, initially the column value should be null.  The value is updated to false (if successfully transferred) or true (if failed)